capi/CMakeLists.txt

```diff
@@ -18,6 +18,12 @@ target_include_directories(geos_c
     $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
     $<INSTALL_INTERFACE:include/geos>)
 
+target_include_directories(geos_c
+  INTERFACE
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
+    $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:include/geos>)
+
```

Comment:
Sorry to drop a review 7 months after the PR, but I see these modifications with the new release. Actually this block is useless and misleading, `geos_c` doesn't have any interface dependency to `geos` headers, just private usage of `geos` headers handled by the link of `geos` target in top CMakeLists.

Actually, everything can be simplified with:

```cmake
target_include_directories(geos_c PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>)
```

and it was already in this CMakeLists

Reply:
It's 7 months later, so I'll need to go and dig it out again... but _something like this_ was clearly needed for it to build/work as a subproject, otherwise it wouldn't have been here 😃 

As a guess, I think because pre-install when building the super-project `source/include/` is needed, since `install/include/geos` isn't populated yet?